services/servers/crypto-pay/types/TInvoice.ts

```diff
@@ -0,0 +1,59 @@
+type TPayUrl = `https://t.me/${'CryptoBot' | 'CryptoTestnetBot'}?start=${TInvoice['hash']}`
+
+export type TInvoice = {
+  invoice_id: number
+  hash: string
+  amount: string
+  fee_asset?: 'USDT' | 'TON' | 'BTC' | 'ETH' | 'LTC' | 'BNB' | 'TRX' | 'USDC' | 'JET' // Available if status is "paid"
```

Comment:
use string

Reply:
Replaces these and followings in the next PR.